.github/actions/build-khiops/action.yml

```diff
@@ -15,6 +15,8 @@ runs:
   steps:
     - name: Setup MPI
       uses: mpi4py/setup-mpi@v1
+    - name: Setup Ninja
```

Comment:
I think you should add `if: ${{ runner.os != 'Linux' }}` because ninja is already on the docker images for Linux

Reply:
OK...